hypex/ab_test/ab_tester.py

```diff
@@ -346,15 +344,15 @@ def sampling_metrics(
             ).pvalue
             t_result[f"{tf} ks-test p-value"] = ks_2samp(ta, tb).pvalue
```

Comment:
there are a lot of repetitions in metric calculations 
it may be worth making a separate function for this